src/concepts/config-intro.md

```diff
@@ -98,11 +98,58 @@ See each of the specific `guard` types linked below for more detailed examples.
 
 ## Vaults
 
-Each configured `vault` represents a container for digital keys and certificates based on a specific implementation `type`.
+Each configured [vault](../reference/config/vaults/) represents a container for digital keys and certificates based on a specific implementation `type`.
 
-Vaults can be used by specific protocol bindings, such as `tls`, to negotiate shared encryption keys.
+### Server Encryption, TLS & SSL
 
-See each of the specific `vault` types linked below for more detailed examples.
+Vaults can be used by specific protocol bindings, such as [tls](../reference/config/bindings/binding-tls.md), to negotiate shared encryption keys. It is easy to add the necessary routing logic and encryption keys.
+
+Using a [filesystem](../reference/config/vaults/vault-filesystem.md) vault, you can see how a pkcs12 certificate on the host is configured to be stored securely by the Zilla runtime. This keystore can then be used by the [tls](../reference/config/bindings/binding-tls.md) binding to decrypt incoming traffic.
+
+```yaml{6}
+vaults:
+  your_server_certificate:
```

Comment:
This seems a bit confusing as the `vault` represents a collection of key + certificate pairs, referencing a specific `key` by name later in `tls` binding.

Reply:
I see what you are saying, since it is a dynamic name I want to encourage using a name specific to the usage. Maybe `your_server_tls` is better? Or `server_tls`?

Reply:
Maybe just name it something like `your_servers`?